net-misc/turbovnc/turbovnc-2.2.5-r2.ebuild

```
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake
+
+DESCRIPTION="A fast replacement for TigerVNC"
+HOMEPAGE="https://www.turbovnc.org/"
+SRC_URI="https://sourceforge.net/projects/turbovnc/files/${PV}/${P}.tar.gz/download -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="virtual/jdk:1.8
+	>=media-libs/libjpeg-turbo-2.0.0[java]
+	!net-misc/tigervnc"
+RDEPEND="${DEPEND}
+	x11-apps/xkbcomp"
+
+src_configure() {
+	local mycmakeargs=(
+		-DTJPEG_JAR=/usr/share/java/turbojpeg.jar
+		-DTJPEG_JNILIBRARY=/usr/lib64/libturbojpeg.so
```

```suggestion
		-DTJPEG_JNILIBRARY="/usr/$(get_libdir)/libturbojpeg.so"
```
so that it works on different architectures